pkg/client/v1/model/initiator_group.go

```diff
@@ -3,33 +3,109 @@
-   // SearchName
-   SearchName string `json:"search_name,omitempty"`
-   // Description
-   Description string `json:"description,omitempty"`
-   // HostType
-   HostType string `json:"host_type,omitempty"`
-   // CreationTime
-   CreationTime float64 `json:"creation_time,omitempty"`
-   // LastModified
-   LastModified float64 `json:"last_modified,omitempty"`
-   // VpOverrIDe
-   VpOverrIDe bool `json:"vp_override,omitempty"`
-   // AppUuID
-   AppUuID string `json:"app_uuid,omitempty"`
-   // VolumeCount
-   VolumeCount float64 `json:"volume_count,omitempty"`
-   // NumConnections
-   NumConnections float64 `json:"num_connections,omitempty"`
+   
+    // Identifier for initiator group.
+    
+ 	ID *string `json:"id,omitempty"`
+   
+    // Name of initiator group.
+    
+ 	Name *string `json:"name,omitempty"`
+   
+    // Initiator group's full name.
+    
+ 	FullName *string `json:"full_name,omitempty"`
+   
+    // Initiator group name used for search.
+    
+ 	SearchName *string `json:"search_name,omitempty"`
+   
+    // Text description of initiator group.
+    
+ 	Description *string `json:"description,omitempty"`
+   
+    // Initiator group access protocol.
+    
+   	AccessProtocol *NsAccessProtocol `json:"access_protocol,omitempty"`
+   
+    // Initiator group host type. Available options are auto and hpux. The default option is auto. This attribute will be applied to all the initiators in the initiator group. Initiators with different host OSes should not be kept in the same initiator group having a non-default host type attribute.
+    
+ 	HostType *string `json:"host_type,omitempty"`
+   
+    // List of target Fibre Channel ports with Target Driven Zoning configured on this initiator group.
+    
+   	FcTdzPorts []*NsFcTdzPort `json:"fc_tdz_ports,omitempty"`
+   
+    // List of target subnet labels. If specified, discovery and access to volumes will be restricted to the specified subnets.
+    
+   	TargetSubnets []*NsTargetSubnet `json:"target_subnets,omitempty"`
+   
+    // List of iSCSI initiators. When create/update iscsi_initiators, either iqn or ip_address is always required with label.
+    
+   	IscsiInitiators []*NsISCSIInitiator `json:"iscsi_initiators,omitempty"`
+   
+    // List of FC initiators. When create/update fc_initiators, wwpn is required.
+    
+   	FcInitiators []*NsFCInitiator `json:"fc_initiators,omitempty"`
+   
+    // Time when this initiator group was created.
+    
+   	CreationTime *int64 `json:"creation_time,omitempty"`
+   
+    // Time when this initiator group was last modified.
+    
+   	LastModified *int64 `json:"last_modified,omitempty"`
+   
+    // Flag to allow modifying VP created initiator groups. When set to true, user can add this initiator to a VP created initiator group.
+    
+ 	VpOverrIDe *bool `json:"vp_override,omitempty"`
+   
+    // Application identifier of initiator group.
+    
+ 	AppUuID *string `json:"app_uuid,omitempty"`
```

Comment:
Also looks like the generator needs some help here.

AppUuID is not correct

Reply:
We have app_uuid under initiator_group which is converted to camel case and id is replaced with ID intentionally. So what exactly is the issue here ?

Reply:
The issue is that id is replaced with ID intentionally.  Look at these 2 examples:
VpOverrIDe should be VpOverride
AppUuID should be AppUuid

We need to handle where id might be used within any field and not just blindly convert it to ID.

Reply:
Ok. will fix it.